applications/eradius_prometheus_collector/src/eradius_prometheus_collector.erl

```diff
@@ -252,18 +247,51 @@ get_metric_info(Name, Stat) ->
+  when Name =:= client_accounting_requests_total; 
+       Name =:= client_accounting_responses ->
+    Type = proplists:get_value(acct_type, Labels),
+    Value = mapping_accounting(Name, Type, Cnt),
     {{ClientName, ClientIP, _ClientPort}, {_, ServerIP, ServerPort}} = Cnt#client_counter.key,
-    Value = element(MetricIdx + 1, Cnt),
     {Value, [{server_name, Cnt#client_counter.server_name}, {server_ip, ServerIP}, {server_port, ServerPort},
-             {client_name, ClientName}, {client_ip, ClientIP}]}.
+           {client_name, ClientName}, {client_ip, ClientIP}, {acct_type, Type}]};
+get_labels_and_val(MetricIdx, {#client_counter{} = Cnt, client}, _) ->
+   {{ClientName, ClientIP, _ClientPort}, {_, ServerIP, ServerPort}} = Cnt#client_counter.key,
+   Value = element(MetricIdx + 1, Cnt),
+   {Value, [{server_name, Cnt#client_counter.server_name}, {server_ip, ServerIP}, {server_port, ServerPort},
+            {client_name, ClientName}, {client_ip, ClientIP}]};
+get_labels_and_val(_, _, _) ->
+    {[], []}.
+
+%% @private
+mapping_accounting(accounting_requests_total, start, #nas_counter{accountRequestsStart = Value})             -> Value;
+mapping_accounting(accounting_requests_total, stop, #nas_counter{accountRequestsStop = Value})               -> Value;
+mapping_accounting(accounting_requests_total, update, #nas_counter{accountRequestsUpdate = Value})           -> Value;
+mapping_accounting(accounting_requests_total, undefined, #nas_counter{accountRequests = Value})              -> Value;
+mapping_accounting(accounting_responses_total, start, #nas_counter{accountResponsesStart = Value})           -> Value;
+mapping_accounting(accounting_responses_total, stop, #nas_counter{accountResponsesStop = Value})             -> Value;
+mapping_accounting(accounting_responses_total, update, #nas_counter{accountResponsesUpdate = Value})         -> Value;
+mapping_accounting(accounting_responses_total, undefined, #nas_counter{accountResponses = Value})            -> Value;
+mapping_accounting(client_accounting_requests_total, start, #client_counter{accountRequestsStart = Value})   -> Value;
+mapping_accounting(client_accounting_requests_total, stop, #client_counter{accountRequestsStop = Value})     -> Value;
+mapping_accounting(client_accounting_requests_total, update, #client_counter{accountRequestsUpdate = Value}) -> Value;
+mapping_accounting(client_accounting_requests_total, undefined, #client_counter{accountRequests = Value})    -> Value;
+mapping_accounting(client_accounting_responses, start, #client_counter{accountResponsesStart = Value})       -> Value;
+mapping_accounting(client_accounting_responses, stop, #client_counter{accountResponsesStop = Value})         -> Value;
+mapping_accounting(client_accounting_responses, update, #client_counter{accountResponsesUpdate = Value})     -> Value;
+mapping_accounting(client_accounting_responses, undefined, #client_counter{accountResponses = Value})        -> Value.
```

Comment:
There is a warning that there is no new line in the end of file, it is not really critical but other files have it